### PR TITLE
Add Crystal.tmbundle

### PR DESCRIPTION
**Pull request by huacnlee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -481,6 +481,8 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
    * [crystal-spacemacs-layer](https://github.com/juanedi/crystal-spacemacs-layer) - Spacemacs contribution layer for Crystal
  * Sublime
    * [sublime-crystal](https://github.com/crystal-lang/sublime-crystal) - Crystal syntax highlighting for sublime Text
+ * TextMate
+   * [Crystal.tmbundle](https://github.com/huacnlee/Crystal.tmbundle) - Crystal syntax highlighting, compile, format command, snippets.
```
Comment (huacnlee) [inline README.md]: full stop?

Comment (huacnlee) [inline README.md]: Oh I see, please just merge and remove it.

Comment (veelenga) [inline README.md]: @huacnlee okay. Fixed and merged. Thanks for contribution

**Comment by huacnlee** [comment]
Thanks. I can't change that because I am going to sleep. (+8:00 Timezone)
-- 
李华顺 (Jason Lee) <http://huacnlee.com>
----------------------------------------
http://huacnlee.com
http://twitter.com/huacnlee
http://github.com/huacnlee

**Pull request merged**